soap-authorization-policy/src/main/apiman/policyDefs/schemas/soap-authorization-PolicyDef.schema

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "SOAP Authorization Policy Configuration",
+  "type": "object",
+  "options": {
+    "disable_collapse": true
+  },
+  "properties": {
+    "rules": {
+      "title": "Add Authorization Rule",
+      "description": "Manage the list of fine-grained authorization rules in the box below. If you want a single required role to protect your entire API, simply add one item with a path of \".*\" and an action of \"*\".",
+      "type": "array",
+      "format": "table",
+      "items": {
+        "type": "object",
+        "title": "Rule",
+        "properties": {
+          "pathPattern": {
+            "title": "Ignored URI",
+            "type": "string"
+          },
+          "soapAction": {
+            "title": "SOAPAction",
+            "type": "string"
+          },
+          "role": {
+            "title": "Role",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "multiMatch": {
+      "title": "Multiple Match Action",
+      "type": "string",
+      "enum": [
+        "I want the request to pass when all matching authorization rules pass.",
+        "I want the request to pass when at least one matching authorization rules pass."
```

Comment:
I think you can add a 'description' property to the 'multiMatch' object.  The description can explain what the values mean.  The UI doesn't need to mirror the existing auth policy UI (which is likely not possible).  But the enum values really do need to reflect the values in the Java enum or else this will not work.